cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -251,14 +254,218 @@ type OpcodeInserter = fn(
+                // https://github.com/bytecodealliance/wasmtime/issues/4313
+                (Opcode::Smax, &[I128, I128]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4313
+                (Opcode::Umax, &[I128, I128]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::Band, &[F32, F32]),
+                (Opcode::Band, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::Bor, &[F32, F32]),
+                (Opcode::Bor, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::Bxor, &[F32, F32]),
+                (Opcode::Bxor, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::Bnot, &[F32, F32]),
+                (Opcode::Bnot, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::BandNot, &[F32, F32]),
+                (Opcode::BandNot, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::BorNot, &[F32, F32]),
+                (Opcode::BorNot, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4870
+                (Opcode::BxorNot, &[F32, F32]),
+                (Opcode::BxorNot, &[F64, F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/5198
+                (Opcode::Bitselect, &[I128, I128, I128]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4934
+                (Opcode::FcvtToUint, &[F32]),
+                (Opcode::FcvtToUint, &[F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4934
+                (Opcode::FcvtToUintSat, &[F32]),
+                (Opcode::FcvtToUintSat, &[F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4934
+                (Opcode::FcvtToSint, &[F32]),
+                (Opcode::FcvtToSint, &[F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4934
+                (Opcode::FcvtToSintSat, &[F32]),
+                (Opcode::FcvtToSintSat, &[F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4933
+                (Opcode::FcvtFromUint, &[I128], &[F32]),
+                (Opcode::FcvtFromUint, &[I128], &[F64]),
+                // https://github.com/bytecodealliance/wasmtime/issues/4933
+                (Opcode::FcvtFromSint, &[I128], &[F32]),
+                (Opcode::FcvtFromSint, &[I128], &[F64]),
+            )
+        }
+
+        _ => true,
+    }
+}
+
+type OpcodeSignature = (
     Opcode,
     &'static [Type], // Args
     &'static [Type], // Rets
     OpcodeInserter,
-)] = &[
+);
+
+// TODO: Derive this from the `cranelift-meta` generator.
+#[rustfmt::skip]
+const OPCODE_SIGNATURES: &'static [OpcodeSignature] = &[
```

Comment:
Unimportant and pre-existing nit: I believe the `'static` lifetime isn't required here, so I'd leave it out.
```suggestion
const OPCODE_SIGNATURES: &[OpcodeSignature] = &[
```